### PR TITLE
Add employee function implementedAdd employee function implemented

### LINKED ISSUE #19: Create function to add employee
# REQUIREMENTS
Create a function to obtain the following data from a form: Name, Employee number, email, position and date of attendance.
The data received by the form should be added to the database in the employees table.

# Acceptance Criteria
- [ ] Function Add Employee
- [ ] Form Add Employee

### PR DESCRIPTION
**Pull request by RobertoCarMed**
The roles that correspond to the employee controller have been created.
Within the create function, what is necessary was implemented to insert a new employee into the database.
The view was also created with the form that corresponds to a new user

These changes were implemented to be able to insert a new employee into the database

Co-authored-by: RobertoCarMed <betocarrasco3652@gmail.com>
Co-authored-by: mrivh <m.alicia.riv@gmail.com>

Resolve #19

### CODE REVIEW

File: app/controllers/employees_controller.rb [hunk 1]
```diff
@@ -5,4 +5,33 @@ class EmployeesController < ApplicationController
...
+  end
+
+  def create
+    data = params.require(:employee)
+    employee = Employee.new
+    employee.nameEmployee = data["nameEmployee"]
+    employee.emailEmployee = data["emailEmployee"]
+    employee.positionEmployee = data["positionEmployee"]
+    employee.attendanceDate = data["attendanceDate"]
+    employee.administrator_id = 1
```
Comment (juliocabrera820) [inline app/controllers/employees_controller.rb:24]: you can define a private method called employee_params and invoke it Employee.new(employee_params)
```ruby
private

def employee_params
  params.require(:employee).permit(:nameEmployee, :emailEmployee, :positionEmployee, :attendance_date, :administrator_id)
end
```

File: app/controllers/employees_controller.rb [hunk 2]
```diff
@@ -5,4 +5,33 @@ class EmployeesController < ApplicationController
...
+
+    #employee.administrator_id = data["1"]
+    #Employee.create(data)
+    #.permit(:employeeNumber, :nameEmployee, :emailEmployee, :positionEmployee, :attendanceDate, :administrator_id)
```
Comment (juliocabrera820) [inline app/controllers/employees_controller.rb:28]: commented-out code must be removed

**Review by juliocabrera820 — APPROVED** [review]
Nice job 👍🏼

**Review by xFerchoVJ — APPROVED** [review]
Everything looks good to me!

**Pull request merged**